tests/test_files_ops.py

```diff
@@ -101,6 +101,127 @@ def test_upload_fails_on_unauthenticated_request(self):
+        self.assertEqual(request_body, responses.calls[0].request.body)
+        self.assertEqual(
+            camel_dict_to_snake_dict(mock_response_metadata),
+            resp.response_metadata.__dict__,
+        )
+        self.assertEqual(url, responses.calls[0].request.url)
+    @responses.activate
     def test_binary_upload_succeeds(self):
```

Comment:
Test case for a non image upload like pdf,  json etc are missing

Let's do 1 image, 1 video, 1 pdf and 1 json file